src/components/Code/Code.tsx

```diff
@@ -25,10 +25,10 @@ export const Code: FunctionComponent<CodeProps> = ({
       {(props) => (
         <Box
           as="pre"
-          border="base"
```

Comment:
Removing the border here makes it difficult to distinguish the singe line code blocks from the document.

<img width="909" alt="Screen Shot 2021-11-09 at 12 57 49 PM" src="https://user-images.githubusercontent.com/8749859/141011633-a014e834-cf00-4550-95ad-de4b192f25fe.png">

This also seems to break the border radius for the multi line code block

<img width="905" alt="Screen Shot 2021-11-09 at 12 57 28 PM" src="https://user-images.githubusercontent.com/8749859/141011741-7792b7ab-8768-4f6c-a416-c207e48f9435.png">